src/Features/Core/Portable/InitializeParameter/AbstractInitializeParameterCodeRefactoringProvider.cs

```diff
@@ -101,12 +101,15 @@ public override async Task ComputeRefactoringsAsync(CodeRefactoringContext conte
             // We support initializing parameters, even when the containing member doesn't have a
             // body. This is useful for when the user is typing a new constructor and hasn't written
             // the body yet.
-            var blockStatementOpt = GetBlockOperation(functionDeclaration, semanticModel, cancellationToken);
-          
+            if (!TryGetBlockOperation(functionDeclaration, semanticModel, cancellationToken, out var blockOperation))
+            {
+                return;
+            };
```

Comment:
1. based on how you changed TryGetBlockOperation, blockOperation should still be called 'opt' since you can return null for it.
2. superfluous semicolon after the if-statement.